com.unity.render-pipelines.universal/Runtime/2D/Light2D.cs

```diff
@@ -414,5 +417,17 @@ public void OnAfterDeserialize()
                 m_ComponentVersion = ComponentVersions.Version_1;
             }
         }
+
+#if UNITY_EDITOR
+        // Handles adding of new layers and removing of invalid layers
+        public void UpdateTargetSortingLayer(SortingLayer layer)
+        {
+            m_ApplyToSortingLayers.RemoveAll(id => !SortingLayer.IsValid(id));
+
+            if (m_ApplyToSortingLayers.Count + 1 == SortingLayer.layers.Length && !m_ApplyToSortingLayers.Contains(layer.id))
```

Comment:
not sure what's going on here... walk me through later?

are you trying to clean up, deleted layes?